projects/04-md-links/README.md

```diff
@@ -189,38 +210,111 @@ Si pasamos la opción `--stats` el output (salida) será un texto con estadísti
 básicas sobre los links.
 
 ```sh
-$ md-links ./some/dir/ --stats
+$ md-links ./some/example.md --stats
 Total: 3
 Unique: 3
-Domains: 3
 ```
 
 También podemos combinar `--stats` y `--validate` para obtener estadísticas que
 necesiten de los resultados de la validación.
 
 ```sh
-$ md-links ./some/dir/ --stats --validate
+$ md-links ./some/example.md --stats --validate
 Total: 3
 Unique: 3
-Domains: 3
 Broken: 1
 ```
 
-## Hacker edition
-
-* Integración continua con Travis o Circle CI
-* Coverage con Coveralls
-* Más estadísticas?
-* Browser compatible (input as string)?
-
 ## Entregables
 
 Módulo instalable via `npm install <github-user>/md-links`. Este módulo debe
 incluir tanto un ejecutable como una interfaz que podamos importar con `require`
 para usarlo programáticamente.
 
+## Hacker edition
+
+- Implementar la librería para que pueda recibir como path(ruta )una carpeta/directorio con diversos archivos y obtengas los links de todos los archivos markdown.
+
+## Pistas / Tips / Recursos
+
+### Pistas
+
+- [Marked](https://github.com/markedjs/marked/blob/master/docs/USING_PRO.md)
```

Comment:
¿Será de verdad una pista agregar marked?
Creo que quedó patente que es más lo que confunde que lo que agrega

Reply:
Teniendo en cuenta los objetivos de aprendizaje del proyecto, creo que sí mencionaría `marked` usando en combinación con un _custom renderer_ (para obtener los `tokens`) y alternativamente la opción de transformar primero el markdown a HTML, y de ahí usar algo tipo JSDOM para "seleccionar" los links dentro de la estructura del DOM.

Eso sí, tal y como está ahorita puede ser más confusión que ayuda...

Qué opinan?

cc/ @developerVilchez @Danielalab @RuthSalvador @merunga @espinozasenior

Reply:
Ayer en el ejercicio de diseño del flujo, cómo extraer los links fue uno de las grandes incognitas. La mayoria propuso regex a pesar de que ya habian tenido mas de una hora para leer y analizar el reto...

Realmente no si confunde o no, pero poniendolo asi de explicito, creo que es cuestion de que una lo descifre para que todas terminen usando _marked_. Lo que podemos intentar es darle una referencia mas generica y con mas contexto sobre parseo de md, o darle las 3 referencias: marked, regexp y dom, para que por lo menos se expongan a la "incomodidad" de tener que decidir.

Reply:
En ese sentido se agregaría al readme como pista las 3 referencias con las cuales se puede hacer el parseo del archivo readme con el objetivo de ver diferentes soluciones implementadas.

Reply:
Creo que finalmente incluiría _sugerencias_ de vías de implementación...

* [markdown-it](https://github.com/markdown-it/markdown-it) como demuestra @merunga [acá](https://github.com/Laboratoria/curricula-js/pull/684#discussion_r209612528)
* `marked` + `JSDOM` o `Cheerio` (ejemplo en mi implementación compartida por Slack)
* `regex`

En un rato mando PR para ilustrar mi propuesta.